src/sqlfluff/core/linter/runner.py

```diff
@@ -211,3 +207,28 @@ def __init__(self, ee):
     def reraise(self):
         """Reraise the encapsulated exception."""
         raise self.ee.with_traceback(self.tb)
+
+
+def get_runner(
+    linter,
+    config,
+    parallel: int,
+    allow_process_parallelism: bool = True,
+) -> BaseRunner:
+    """Generate a runner instance based on parallel and sytem configuration."""
+    if parallel > 1 and sys.version_info > (3, 7):
```

Comment:
I'm thinking now that the version check should use `>=`, i.e.: `sys.version_info >= (3, 7)` because it's 3.6.x that is not supporting parallel.

Reply:
I've updated the comparison. Do you have something I can link to and put in a comment about *why* 3.6 isn't supported. I had a quick google just now and couldn't work it out.

Reply:
TBH, I don't remember. I know it wasn't working. I recall thinking we could possibly get it working, but 3.6 is so old, was it worth the effort.

I'll see if I can reproduce the error on 3.6 again.